File: js/programmers/level.1/prgm_155652_1.js
function solution(s, skip, index) {
  let answer = '';

  for (const i of s) {
    let code = i.charCodeAt();

    for (let j = 0; j < index; j++) {
      code += 1;
      if (code > 122) code = 97;

      while (skip.includes(String.fromCharCode(code))) {
        code += 1;
        if (code > 122) code = 97;
      }
    }

    answer += String.fromCharCode(code);
  }

  return answer;
}

File: python/programmers/level.1/prgm_155652_1.py
def solution(s, skip, index):
    answer = ""

    for i in s:
        code = ord(i)

        for j in range(index):
            code += 1
            if code > 122: code = 97

            while chr(code) in skip:
                code += 1
                if code > 122: code = 97

        answer += chr(code)

    return answer